lib/Popup.js

```diff
@@ -124,7 +124,10 @@ exports.default = _react2.default.createClass({
 
     if (opening) this.open();else if (closing) this.close();else if (open) {
       var height = this.height();
-      if (height !== this.state.height) this.setState({ height: height });
+      var diff = Math.abs(height - this.state.height);
+      if (isNaN(diff) || diff > 0.1) {
```

Comment:
can you add a comment describing what this is fixing so it doesn't get removed later accidentally?